test/jaxmd.py

```diff
@@ -1,55 +1,26 @@
 from absl.testing import absltest
-import jax.numpy as jnp
-import jax.random
-import jax.lax
-import enzyme_ad.jax as enzyme_jax
-from enzyme_ad.jax import (
-    enzyme_jax_ir,
-    NewXLAPipeline,
-    OldXLAPipeline,
-    JaXPipeline,
-    hlo_opts,
-)
-import numpy as np
-import timeit
-from test_utils import *
 
-argv = ("-I/usr/include/c++/11", "-I/usr/include/x86_64-linux-gnu/c++/11")
+from test_utils import *
 
-import jax.numpy as np
-import numpy as onp
-from jax import jit
-from jax import random
-from jax import lax
 
-jax.config.update("jax_enable_x64", True)
+class JAXMD(EnzymeJaxTest):
+    def setUp(self):
 
-pipelines = [
-    ("JaX  ", None, CurBackends),
-    ("JaXPipe", JaXPipeline(), CurBackends),
-    (
-        "HLOOpt",
-        JaXPipeline(
-            "inline{default-pipeline=canonicalize max-iterations=4},"
-            + "canonicalize,cse,enzyme-hlo-opt,cse"
-        ),
-        CurBackends,
-    ),
-    ("PartOpt", JaXPipeline(partialopt), CurBackends),
-    ("DefOpt", JaXPipeline(hlo_opts()), CurBackends),
-]
+        from jax import jit
+        from jax import random
+        from jax import lax
 
+        import jax.numpy as np
 
-class JAXMD(EnzymeJaxTest):
-    def setUp(self):
         from jax_md import space
         from jax_md import energy
         from jax_md import simulate
         from jax_md import quantity
         from jax_md import partition
 
         lattice_constant = 1.37820
-        N_rep = 40
+        # We hit a GPU memory limit for N_rep = 40
+        N_rep = 10 if jax.default_backend() == "gpu" else 40
```

Comment:
Should we just keep 10 so we don't accidentally compare mismatching CPU/GPU later?